common/utils.go

```diff
@@ -152,3 +152,11 @@ func GetCiliumVersionString(epCHeaderFilePath string) (string, error) {
 		}
 	}
 }
+
+// RequireRootPrivilege checks if the user running cmd is root. If not, it exits the program
+func RequireRootPrivilege(cmd string) {
+	if os.Getuid() != 0 {
+		fmt.Fprintf(os.Stderr, "Please run %q commands with root privileges.\n", cmd)
```

Comment:
I'm fine with having it inside. We also fail at various places already with `log.Fatalf()`